common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -558,6 +557,12 @@ def role(self):
         }
         return roles.get(self.system.get_user_role(), u'Student')
 
+    @property
+    def icon_class(self):
```

Comment:
Might be one less step to override `get_icon_class`? But both work.

Reply:
Ah okay, and that's the part that comes from the shims? I wasn't sure which would be preferred.

Reply:
I think they both come from shim. And the `icon_class` variable is just a convenience so that xblocks can have a one liner to set icon.

Reply:
Cool, updating to `get_icon_class`